### PR TITLE
Completed node-notes-cli. | Rakesh

### PR DESCRIPTION
**Pull request by bhaisingh**
image

### CODE REVIEW

File: node-notes-cli/createnote.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            fs.writeFile(FileName, jsonString, err => {
+                if (err) {
+                    console.log('Error writing JSON File: ', err);
+                }    
+            })
+        } catch (err) {
+            console.log("Error parsing JSON string: ", err);
+        } 
+    });
+}
```
Comment (TimelikeClosure) [inline node-notes-cli/createnote.js:25]: How would you prevent the user from accidentally adding an empty note?

File: node-notes-cli/deletenote.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            fs.writeFile(FileName, jsonString, err => {
+                if (err) {
+                    console.log('Error writing JSON File: ', err);
+                }    
+            })
+        } catch (err) {
+            console.log("Error parsing JSON string: ", err);
+        } 
+    });
+}
```
Comment (TimelikeClosure) [inline node-notes-cli/deletenote.js:23]: How would you let the user know if they tried to delete a note that doesn't exist?

File: node-notes-cli/updatenote.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+            fs.writeFile(FileName, jsonString, err => {
+                if (err) {
+                    console.log('Error writing JSON File: ', err);
+                }    
+            })
+        } catch (err) {
+            console.log("Error parsing JSON string: ", err);
+        } 
+    });
+}
```
Comment (TimelikeClosure) [inline node-notes-cli/updatenote.js:23]: "User can update a note" shouldn't allow the user to create a new note by accident. How would you prevent that from happening?

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: node-notes-cli/updatenote.js (hunk 3 above)
Comment (bhaisingh) [reply node-notes-cli/updatenote.js]: Fixed validation issue.
image

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**